pyuvdata/utils.py

```diff
@@ -115,6 +120,13 @@
                     "rr": ["r", "r"], "ll": ["l", "l"],
                     "rl": ["r", "l"], "lr": ["l", "r"]}
 
+ANGLE_TIME_EQUIV = [(units.s, units.arcsec, lambda x: x * 15.0, lambda x: x / 15.0)]
```

Comment:
Maybe I'm missing it, but I don't see anywhere this is used...

Reply:
Ah, that's a bit of leftover code from when I was originally coding this. It's been removed.